pkg/agent/openflow/pipeline.go

```diff
@@ -305,8 +306,9 @@ func GetAntreaPolicyMultiTierTables() []*Table {
 }
 
 const (
-	CtZone       = 0xfff0
-	CtZoneV6     = 0xffe6
+	// Connection Tracking Zones for traffic without an 802.1Q header
+	CtZone       = 0x1000
+	CtZoneV6     = 0x3000
```

Comment:
Does it mean all existing connection will be broken once restart even for NodeIPAM Pod?
In theory, when antrea-agent restarted, there might be packet drops but connections could continue working soon.
Is it a must to change CT zone for non-VLAN case?

Reply:
For AntreaIPAM enabled, it is a must to change CT zone since we need to add VLAN ID into CT zone.
For AntreaIPAM disabled, it is possible to keep existing CT zone value.
Do you think we should change it back?

Reply:
Yes, we should guarantee graceful restart/upgrade as best we can. I confirmed established connections can continue working after restart/upgrade previously, and hard to say users are not expecting it. So please keep original zone for disabled case.

Reply:
OK. Reverted this change and re-run e2e.